### ISSUE TITLE
System Preferences Enabled/Disabled/Hidden Preference Panes

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileManifests#66]
In the main System Preferences payload, it's currently possible to configure any preference pane to be simultaneously enabled, disabled, and hidden, which I'm guessing would cause problems if done in a profile.

![Screen Shot 2019-03-20 at 10 15 57 AM](https://user-images.githubusercontent.com/11789931/54691197-3485b200-4af9-11e9-8fba-3b7cf15a79f7.png)
![Screen Shot 2019-03-20 at 10 16 00 AM](https://user-images.githubusercontent.com/11789931/54691215-38b1cf80-4af9-11e9-896b-e1fb7c78c0ba.png)
![Screen Shot 2019-03-20 at 10 16 05 AM](https://user-images.githubusercontent.com/11789931/54691221-3b142980-4af9-11e9-986f-ce7c77ecae0c.png)

It would be helpful once one of these preference panes was configured in one of these areas that it became greyed out and inaccessible from the others to avoid unintended duplicates.  

Unsure at the moment about how this could be accomplished ...

**Comment by erikberglund** [comment]
Good catch. Another thing about this payload is that it should allow the admin to enter custom preference pane identifiers in an array to also add to one of the enabled, disabled etc. That is also an issue to look at. All of these things require code changes though.
